affected_purls:
- pkg:pypi/pykmip@0.0.1
- pkg:pypi/pykmip@0.1.0
- pkg:pypi/pykmip@0.1.1
- pkg:pypi/pykmip@0.2.0
- pkg:pypi/pykmip@0.3.0
- pkg:pypi/pykmip@0.3.1
- pkg:pypi/pykmip@0.3.2
- pkg:pypi/pykmip@0.3.3
- pkg:pypi/pykmip@0.4.0
- pkg:pypi/pykmip@0.4.1
- pkg:pypi/pykmip@0.5.0
- pkg:pypi/pykmip@0.6.0
- pkg:pypi/pykmip@0.7.0
aliases:
- CVE-2018-1000872
- GHSA-gfv6-cj92-g3hx
fixed_purl:
- pkg:deb/debian/python-pykmip@0.10.0-3?distro=sid
- pkg:deb/debian/python-pykmip@0.10.0-4?distro=sid
- pkg:deb/debian/python-pykmip@0.7.0-3?distro=sid
- pkg:pypi/pykmip@0.8.0
references:
- id: 143557
  reference_id: ''
  url: https://github.com/OpenKMIP/PyKMIP/commit/3a7b880bdf70d295ed8af3a5880bab65fa6b3932
- id: 143555
  reference_id: ''
  url: https://github.com/OpenKMIP/PyKMIP/issues/430
- id: 127736
  reference_id: '917030'
  url: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=917030
- id: 143554
  reference_id: CVE-2018-1000872
  url: https://nvd.nist.gov/vuln/detail/CVE-2018-1000872
- id: 143556
  reference_id: GHSA-gfv6-cj92-g3hx
  url: https://github.com/advisories/GHSA-gfv6-cj92-g3hx
severities:
- id: 20027
  reference_id: 143556
  scoring_elements: ''
  scoring_system: cvssv3.1_qr
  value: MODERATE
summary: 'OpenKMIP PyKMIP version All versions before 0.8.0 contains a CWE 399: Resource
  Management Errors (similar issue to CVE-2015-5262) vulnerability in PyKMIP server
  that can result in DOS: the server can be made unavailable by one or more clients
  opening all of the available sockets. This attack appear to be exploitable via A
  client or clients open sockets with the server and then never close them. This vulnerability
  appears to have been fixed in 0.8.0.'
vulnerability_id: VCID-3bm9-fhtp-aaam
weaknesses: []
